webapp/channels/src/components/claim/components/oauth_to_email.tsx

```diff
@@ -134,4 +133,4 @@ const OAuthToEmail = (props: Props) => {
     );
 };
 
-export default OAuthToEmail;
+export default injectIntl(OAuthToEmail);
```

Comment:
Since this is a functional component we can use the hook directly instead of this hoc https://formatjs.io/docs/react-intl/api/#useintl-hook

Reply:
@M-ZubairAhmed changed

Reply:
I still dont see the hook `useIntl`? How are you accessing the intl prop?

Reply:
refer https://github.com/mattermost/mattermost/pull/24839#event-10593438518